### PR TITLE
misc(gulp): rename output files

### PR DESCRIPTION
**Pull request by wardpeet**
**Summary**
renaming consumer's files:
`devtools-entry.js => lighthouse-dt-bundle.js`
`extension-entry.js => lighthouse-ext-bundle.js`
`lightrider-entry.js => lighthouse-lr-bundle.js`
 
also removed debug from browserify which I accidentally added.

```bash
$ gulp build:production
[18:13:07] Using gulpfile /c/Users/WardPeeters/Projects/os/lighthouse/lighthouse-extension/gulpfile.js
[18:13:07] Starting 'build:production'...
[18:13:07] Starting 'build'...
[18:13:07] Starting 'lint'...
[18:13:07] Finished 'lint' after 647 ms
[18:13:07] Starting 'browserify'...
[18:13:07] Starting 'browserify-lighthouse'...
[18:13:07]  app/src/devtools-entry.js
[18:13:07] renamed into: app/src/lighthouse-dt-bundle.js
[18:13:07]  app/src/extension-entry.js
[18:13:07] renamed into: app/src/lighthouse-ext-bundle.js
[18:13:07]  app/src/lightrider-entry.js
[18:13:07] renamed into: app/src/lighthouse-lr-bundle.js
[18:13:07]  3 items
[18:13:07] renamed into: 3 items
[18:13:15] Finished 'browserify-lighthouse' after 7.67 s
[18:13:15] Starting 'browserify-other'...
[18:13:15] Finished 'browserify-other' after 86 ms
[18:13:15] Finished 'browserify' after 7.76 s
[18:13:15] Starting 'chromeManifest'...
[18:13:15] Finished 'chromeManifest' after 166 ms
[18:13:15] Starting 'html'...
[18:13:15] Starting 'images'...
[18:13:15] Starting 'css'...
[18:13:15] Starting 'extras'...
[18:13:15] copying to dist: app/_locales
[18:13:15] copying to dist: app/_locales/en
[18:13:15] copying to dist: app/_locales/en/messages.json
[18:13:15] copying to dist: app/_locales/en/messages_canary.json
[18:13:15] copying to dist: 4 items
[18:13:15] Finished 'html' after 19 ms
[18:13:15] Finished 'css' after 36 ms
[18:13:15] Finished 'extras' after 41 ms
[18:13:15] Finished 'images' after 60 ms
[18:13:15] Finished 'build' after 8.64 s
[18:13:15] Starting 'compilejs'...
[18:13:24] Finished 'compilejs' after 8.33 s
[18:13:24] Finished 'build:production' after 17 s
```

sizes are intact, there is a small diff but it's just whitespace
```bash
-rw-rw-rw- 1 0 0 1354774 Oct  5 18:28 lighthouse-background.js
-rw-rw-rw- 1 0 0 1354379 Oct  5 18:27 lighthouse-dt-bundle.js
-rw-rw-rw- 1 0 0 1525837 Oct  5 18:28 lighthouse-ext-background.js
-rw-rw-rw- 1 0 0 1525425 Oct  5 18:27 lighthouse-ext-bundle.js
-rw-rw-rw- 1 0 0 2568125 Oct  5 18:28 lighthouse-lr-background.js
-rw-rw-rw- 1 0 0 2567727 Oct  5 18:27 lighthouse-lr-bundle.js
```

**Related Issues/PRs**

#6170

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -157,7 +157,15 @@
   },
   "bundlesize": [
     {
-      "path": "./lighthouse-extension/dist/scripts/lighthouse-background.js",
+      "path": "./lighthouse-extension/dist/scripts/lighthouse-dt-bundle.js",
```
Comment (wardpeet) [inline package.json]: unsure which one to check or if we should check them all 3

**Comment by wardpeet** [comment]
I tried using gulp-rename but it did not copy my whole file for some reason so I switched to a tap instance.

File: docs/releasing.md [hunk 2]
```diff
@@ -93,8 +93,8 @@ git push --tags
 echo "Rebuild extension and viewer to get the latest, tagged master commit"
 yarn build-all;
 
-# zip the extension files, but remove lh-background as it's not needed
-cd lighthouse-extension; command rm -f dist/scripts/lighthouse-background.js; gulp package; cd ..
```
Comment (brendankenny) [inline docs/releasing.md]: we should really update `gulp package` to do this for us. I've definitely never remembered to delete the other files :):)

File: lighthouse-extension/gulpfile.js [hunk 3]
```diff
@@ -60,8 +70,10 @@ const computedArtifacts = LighthouseRunner.getComputedGathererList()
 const locales = fs.readdirSync('../lighthouse-core/lib/i18n/locales/')
     .map(f => require.resolve(`../lighthouse-core/lib/i18n/locales/${f}`));
 
-const isDevtools = file => file.endsWith(CONSUMERS.DEVTOOLS);
-const isExtension = file => file.endsWith(CONSUMERS.EXTENSION);
+const isDevtools = file =>
+  file.endsWith(CONSUMERS.DEVTOOLS.src) || file.endsWith(CONSUMERS.DEVTOOLS.dist);
```
Comment (brendankenny) [inline lighthouse-extension/gulpfile.js]: can the `dist` file really ever go through this function?

File: lighthouse-extension/test/app/srclightrider-entry.js [hunk 4]
```diff
@@ -6,12 +6,12 @@
 'use strict';
```
Comment (brendankenny) [inline lighthouse-extension/test/app/srclightrider-entry.js]: I think you missed a `/` in the `git mv` :)

File: lighthouse-extension/app/src/extension-entry.js [hunk 5]
```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 const lighthouse = require('../../../lighthouse-core/index');
-const background = require('./lighthouse-background');
+const background = require('./devtools-entry');
```
Comment (brendankenny) [inline lighthouse-extension/app/src/extension-entry.js]: it's unfortunate to have to keep calling this `background`, but I can't think of a better name right now :)

File: lighthouse-extension/test/app/srclightrider-entry.js (hunk 4 above)
Comment (wardpeet) [reply lighthouse-extension/test/app/srclightrider-entry.js]: oh nice thanks

File: lighthouse-extension/gulpfile.js (hunk 3 above)
Comment (wardpeet) [reply lighthouse-extension/gulpfile.js]: probably not but wanted to be save but i'm removing it

File: lighthouse-extension/app/src/extension-entry.js (hunk 5 above)
Comment (wardpeet) [reply lighthouse-extension/app/src/extension-entry.js]: if I move `getDefaultConfigForCategories`  & `getDefaultCategories` to it's own file called one of them below I could rename the variable which gives more meaning to it.
- `config.js`,
- `helpers/config.js`
- `runOptions.js`
- `lighthouseOptions.js`
- ...

File: docs/releasing.md (hunk 2 above)
Comment (wardpeet) [reply docs/releasing.md]: i'll add a new PR that fixes this.

File: lighthouse-extension/app/src/extension-entry.js (hunk 5 above)
Comment (brendankenny) [reply lighthouse-extension/app/src/extension-entry.js]: yeah, I think that makes sense, but I think we should just wait on it (at least until LR shakes out and we can fully see the shape of these files for the medium-term future).

File: docs/releasing.md (hunk 2 above)
Comment (wardpeet) [reply docs/releasing.md]: @brendankenny see #6210

**Pull request merged**